frontend/ui/widget/confirmbox.lua

```diff
@@ -76,62 +76,73 @@ function ConfirmBox:init()
             self.key_events.Close = { { Device.input.group.Back } }
         end
     end
-    local text_widget = TextBoxWidget:new{
+
+    -- Named so for consistency with other widgets that use addWidget()
+    self._input_widget = TextBoxWidget:new{
```

Comment:
I'm not certain naming this non-input widget `_input_widget` is really better :)
If there is now more that InputDialog that we need to fetch the width so added widget can size themselves, I would just add a new method to InputDialog and ConfirmBox: `:getAddedWidgetAvailableWidth()` and use it.